moveit_core/planning_scene/CMakeLists.txt

```diff
@@ -1,30 +1,56 @@
 set(MOVEIT_LIB_NAME moveit_planning_scene)
 
-add_library(${MOVEIT_LIB_NAME} src/planning_scene.cpp)
+if(WIN32)
+  # set(append_library_dirs "$<TARGET_FILE_DIR:${PROJECT_NAME}>;$<TARGET_FILE_DIR:${PROJECT_NAME}_TestPlugins1>")
```

Comment:
This should be fixed, commented or removed.

Reply:
Done ;)